x/multi-staking/keeper/abci_test.go

```diff
@@ -0,0 +1,69 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	"github.com/realio-tech/multi-staking-module/testutil"
+	"github.com/realio-tech/multi-staking-module/x/multi-staking/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func (suite *KeeperTestSuite) TestEndBlocker() {
+	suite.SetupTest()
+
+	mulStaker := testutil.GenAddress()
+	valAddr := testutil.GenValAddress()
+	const gasDenom = "ario"
```

Comment:
this variable could not be const and you can change this variable name to `multiStakingDenomA`